INFO: Index: 0
Name: Export all activities
Description: Makes all app activities exportable.
Enabled: false

Index: 1
Name: Spoof build info
Description: Spoof the information about the current build.
Enabled: false

Index: 2
Name: Hide mock location
Description: Prevents the app from knowing the device location is being mocked by a third party app.
Enabled: false

Index: 3
Name: Spoof SIM country
Description: Spoofs country information returned by the SIM card provider.
Enabled: false

Index: 4
Name: Spoof Wi-Fi connection
Description: Spoofs an existing Wi-Fi connection.
Enabled: false

Index: 5
Name: Enable Android debugging
Description: Enables Android debugging capabilities. This can slow down the app.
Enabled: false

Index: 6
Name: Export internal data documents provider
Description: Exports a documents provider that grants access to the internal data directory of this app to file managers and other apps that support the Storage Access Framework.
Enabled: false

Index: 7
Name: Hex
Description: Replaces a hexadecimal patterns of bytes of files in an APK.
Enabled: false

Index: 8
Name: Predictive back gesture
Description: Enables the predictive back gesture introduced on Android 13.
Enabled: false

Index: 9
Name: Override certificate pinning
Description: Overrides certificate pinning, allowing to inspect traffic via a proxy.
Enabled: false

Index: 10
Name: Change package name
Description: Appends ".revanced" to the package name by default. Changing the package name of the app can lead to unexpected issues.
Enabled: false

Index: 11
Name: Remove screen capture restriction
Description: Removes the restriction of capturing audio from apps that normally wouldn't allow it.
Enabled: false

Index: 12
Name: Remove screenshot restriction
Description: Removes the restriction of taking screenshots in apps that normally wouldn't allow it.
Enabled: false

Index: 13
Name: Remove share targets
Description: Removes share targets like directly sharing to a frequent contact.
Enabled: false

Index: 14
Name: Change version code
Description: Changes the version code of the app. By default the highest version code is set. This allows older versions of an app to be installed if their version code is set to the same or a higher value and can stop app stores to update the app.
Enabled: false

Index: 15
Name: Always allow deep-linking
Description: Open Amazon links, even if the app is not set to handle Amazon links.
Enabled: true

Index: 16
Name: Pro unlock
Description: null
Enabled: true

Index: 17
Name: Remove play limits
Description: Disables purchase nagging and playback limits of not purchased tracks.
Enabled: true

Index: 18
Name: Bypass root checks
Description: Removes the restriction to use the app with root permissions or on a custom ROM.
Enabled: true

Index: 19
Name: Disable ads
Description: null
Enabled: true

Index: 20
Name: Enable debug menu
Description: null
Enabled: false

Index: 21
Name: Hide 'Sponsored Stories'
Description: null
Enabled: true

Index: 22
Name: Hide story ads
Description: Hides the ads in the Facebook app stories.
Enabled: true

Index: 23
Name: Remove bootloader detection
Description: Removes the check for an unlocked bootloader.
Enabled: true

Index: 24
Name: Remove root detection
Description: Removes the check for root permissions.
Enabled: true

Index: 25
Name: Enable CustomTabs
Description: Enables CustomTabs to open articles in your default browser.
Enabled: true

Index: 26
Name: GmsCore support
Description: Allows patched Google apps to run without root and under a different package name by using GmsCore instead of Google Play Services.
Enabled: true

Index: 27
Name: Spoof features
Description: Spoofs the device to enable Google Pixel exclusive features, including unlimited storage.
Enabled: true

Index: 28
Name: GmsCore support
Description: Allows patched Google apps to run without root and under a different package name by using GmsCore instead of Google Play Services.
Enabled: true

Index: 29
Name: Restore hidden 'Back up while charging' toggle
Description: Restores a hidden toggle to only run backups when the device is charging.
Enabled: true

Index: 30
Name: Remove device restrictions
Description: Removes restrictions from using the app on any device. Requires mounting patched app over original.
Enabled: true

Index: 31
Name: Disable ads
Description: null
Enabled: true

Index: 32
Name: Unlock pro
Description: null
Enabled: true

Index: 33
Name: Remove root detection
Description: Removes the check for root permissions and unlocked bootloader.
Enabled: true

Index: 34
Name: Spoof signature
Description: Spoofs the signature of the app.
Enabled: true

Index: 35
Name: Hide ads
Description: null
Enabled: true

Index: 36
Name: Hide ads
Description: Hides ads in stories, discover, profile, etc. An ad can still appear once when refreshing the home feed.
Enabled: true

Index: 37
Name: Remove ads
Description: null
Enabled: true

Index: 38
Name: Disable mandatory login
Description: null
Enabled: true

Index: 39
Name: Unlock premium
Description: null
Enabled: true

Index: 40
Name: Unlock pro
Description: null
Enabled: true

Index: 41
Name: Hide inbox ads
Description: Hides ads in inbox.
Enabled: true

Index: 42
Name: Hide inbox subtabs
Description: Hides Home and Channels tabs between active now tray and chats.
Enabled: true

Index: 43
Name: Disable switching emoji to sticker
Description: Disables switching from emoji to sticker search mode in message input field.
Enabled: true

Index: 44
Name: Disable typing indicator
Description: Disables the indicator while typing a message.
Enabled: true

Index: 45
Name: Force English locale
Description: Forces wearable devices to use the English locale.
Enabled: true

Index: 46
Name: Fix login
Description: Fixes login for uncertified Mi Fitness app
Enabled: true

Index: 47
Name: Hide music video ads
Description: Hides ads that appear while listening to or streaming music videos, podcasts, or songs.
Enabled: true

Index: 48
Name: Enable exclusive audio playback
Description: Enables the option to play audio without video.
Enabled: true

Index: 49
Name: Permanent repeat
Description: Permanently remember your repeating preference even if the playlist ends or another track is played.
Enabled: false

Index: 50
Name: Hide category bar
Description: Hides the category bar at the top of the homepage.
Enabled: false

Index: 51
Name: Hide 'Get Music Premium' label
Description: Hides the "Get Music Premium" label from the account menu and settings.
Enabled: true

Index: 52
Name: Remove upgrade button
Description: Removes the upgrade tab from the pivot bar.
Enabled: true

Index: 53
Name: Bypass certificate checks
Description: Bypasses certificate checks which prevent YouTube Music from working on Android Auto.
Enabled: true

Index: 54
Name: Remove background playback restrictions
Description: Removes restrictions on background playback, including playing kids videos in the background.
Enabled: true

Index: 55
Name: GmsCore support
Description: Allows patched Google apps to run without root and under a different package name by using GmsCore instead of Google Play Services.
Enabled: true

Index: 56
Name: Spoof client
Description: Spoofs the client to fix playback.
Enabled: true

Index: 57
Name: Unlock pro
Description: null
Enabled: true

Index: 58
Name: Hide ads
Description: Hides most of the ads across the app.
Enabled: true

Index: 59
Name: Remove broadcasts restriction
Description: Enables starting/stopping NetGuard via broadcasts.
Enabled: true

Index: 60
Name: Unlock pro
Description: null
Enabled: true

Index: 61
Name: Fix crash
Description: null
Enabled: true

Index: 62
Name: Spoof device ID
Description: Spoofs device ID to mitigate manual bans by developers.
Enabled: true

Index: 63
Name: Hide update popup
Description: Prevents the update popup from showing up.
Enabled: true

Index: 64
Name: Unlock plus
Description: null
Enabled: true

Index: 65
Name: Spoof Android device ID
Description: Spoofs the Android device ID used by the app for account authentication.This can be used to copy the account to another device.
Enabled: false

Index: 66
Name: Disable tracking
Description: Disables tracking by replacing tracking URLs with example.com.
Enabled: true

Index: 67
Name: Hide ads
Description: null
Enabled: true

Index: 68
Name: Hide purchase reminder
Description: Hides the popup that reminds you to purchase the app.
Enabled: true

Index: 69
Name: Hide ads
Description: null
Enabled: true

Index: 70
Name: Spoof client
Description: Restores functionality of the app by using custom client ID.
Enabled: true

Index: 71
Name: Disable ads
Description: null
Enabled: true

Index: 72
Name: Spoof client
Description: Restores functionality of the app by using custom client ID.
Enabled: true

Index: 73
Name: Fix missing audio in video downloads
Description: Fixes audio missing in videos downloaded from v.redd.it.
Enabled: true

Index: 74
Name: Fix /s/ links
Description: null
Enabled: true

Index: 75
Name: Spoof client
Description: Restores functionality of the app by using custom client ID.
Enabled: true

Index: 76
Name: Unlock subscription
Description: Unlocks the subscription feature but requires a custom client ID.
Enabled: true

Index: 77
Name: Disable ads
Description: null
Enabled: true

Index: 78
Name: Spoof client
Description: Restores functionality of the app by using custom client ID.
Enabled: true

Index: 79
Name: Spoof client
Description: Restores functionality of the app by using custom client ID.
Enabled: true

Index: 80
Name: Spoof client
Description: Restores functionality of the app by using custom client ID.
Enabled: true

Index: 81
Name: Spoof client
Description: Restores functionality of the app by using custom client ID.
Enabled: true

Index: 82
Name: Disable ads
Description: null
Enabled: true

Index: 83
Name: Disable ads
Description: null
Enabled: true

Index: 84
Name: Disable Sync for Lemmy bottom sheet
Description: Disables the bottom sheet at the startup that asks you to signup to "Sync for Lemmy".
Enabled: true

Index: 85
Name: Spoof client
Description: Restores functionality of the app by using custom client ID.
Enabled: true

Index: 86
Name: Fix /s/ links
Description: null
Enabled: true

Index: 87
Name: Use /user/ endpoint
Description: Replaces the deprecated endpoint for viewing user profiles /u with /user, that used to fix a bug.
Enabled: false

Index: 88
Name: Fix video downloads
Description: Fixes a bug in Sync's MPD parser resulting in only the audio-track being saved.
Enabled: true

Index: 89
Name: Disable screenshot popup
Description: Disables the popup that shows up when taking a screenshot.
Enabled: true

Index: 90
Name: Unlock premium Reddit icons
Description: Unlocks the premium Reddit icons.
Enabled: true

Index: 91
Name: Sanitize sharing links
Description: Removes (tracking) query parameters from the URLs when sharing links.
Enabled: true

Index: 92
Name: Remove root detection
Description: Removes the check for root permissions and unlocked bootloader.
Enabled: true

Index: 93
Name: Remove file size limit
Description: Allows opening files larger than 2 MB in the text editor.
Enabled: true

Index: 94
Name: Remove badge tab
Description: Removes the badge tab from the activity tab.
Enabled: true

Index: 95
Name: Remove notification badge
Description: Removes the red notification badge from the activity tab.
Enabled: true

Index: 96
Name: Hide ads
Description: null
Enabled: true

Index: 97
Name: Disable telemetry
Description: Disables SoundCloud's telemetry system.
Enabled: true

Index: 98
Name: Enable offline sync
Description: null
Enabled: true

Index: 99
Name: Custom theme
Description: Applies a custom theme.
Enabled: true

Index: 100
Name: Enable on demand
Description: Enables listening to songs on-demand, allowing to play any song from playlists, albums or artists without limitations. This does not remove ads.
Enabled: true

Index: 101
Name: Premium navbar tab
Description: Hides the premium tab from the navigation bar.
Enabled: true

Index: 102
Name: Hide offers tab
Description: null
Enabled: true

Index: 103
Name: Hide story bubbles
Description: null
Enabled: true

Index: 104
Name: Unlock subscription features
Description: Unlocks "Routes", "Matched Runs" and "Segment Efforts".
Enabled: true

Index: 105
Name: Disable subscription suggestions
Description: null
Enabled: true

Index: 106
Name: Remove Google Play Integrity check
Description: Removes the Google Play Integrity check. With this it's possible to use SwissID on custom ROMS.If the device is rooted, root permissions must be hidden from the app.
Enabled: true

Index: 107
Name: Unlock themes
Description: Unlocks all themes that are inaccessible until a certain level is reached.
Enabled: true

Index: 108
Name: Feed filter
Description: Removes ads, livestreams, stories, image videos and videos with a specific amount of views or likes from the feed.
Enabled: true

Index: 109
Name: Remember clear display
Description: Remembers the clear display configurations in between videos.
Enabled: true

Index: 110
Name: Downloads
Description: Removes download restrictions and changes the default path to download to.
Enabled: true

Index: 111
Name: Show seekbar
Description: Shows progress bar for all video.
Enabled: true

Index: 112
Name: Playback speed
Description: Enables the playback speed option for all videos and retains the speed configurations in between videos.
Enabled: true

Index: 113
Name: Disable login requirement
Description: null
Enabled: true

Index: 114
Name: Fix Google login
Description: Allows logging in with a Google account.
Enabled: true

Index: 115
Name: Settings
Description: Adds ReVanced settings to TikTok.
Enabled: true

Index: 116
Name: SIM spoof
Description: Spoofs the information which is retrieved from the SIM card.
Enabled: false

Index: 117
Name: Unlock pro
Description: null
Enabled: true

Index: 118
Name: Show on lockscreen
Description: Shows student id and student ticket on lockscreen.
Enabled: true

Index: 119
Name: Disable dashboard ads
Description: Disables ads in the dashboard.
Enabled: true

Index: 120
Name: Disable Ad-Free Banner
Description: Disables the banner with a frog, prompting you to buy Tumblr Ad-Free.
Enabled: true

Index: 121
Name: Disable in-app update
Description: Disables the in-app update check and update prompt.
Enabled: true

Index: 122
Name: Disable blog notification reminder
Description: Disables the reminder to enable notifications for blogs you visit.
Enabled: true

Index: 123
Name: Disable gift message popup
Description: Disables the popup suggesting to buy TumblrMart items for other people.
Enabled: true

Index: 124
Name: Fix old versions
Description: Fixes old versions of the app (v33.2 and earlier) breaking due to Tumblr removing remnants of Tumblr Live from the API, which causes many requests to fail. This patch has no effect on newer versions of the app.
Enabled: false

Index: 125
Name: Block audio ads
Description: Blocks audio ads in streams and VODs.
Enabled: true

Index: 126
Name: Block embedded ads
Description: Blocks embedded stream ads using services like Luminous or PurpleAdBlocker.
Enabled: true

Index: 127
Name: Block video ads
Description: Blocks video ads in streams and VODs.
Enabled: true

Index: 128
Name: Show deleted messages
Description: Shows deleted chat messages behind a clickable spoiler.
Enabled: true

Index: 129
Name: Auto claim channel points
Description: Automatically claim Channel Points.
Enabled: true

Index: 130
Name: Debug mode
Description: Enables Twitch's internal debugging mode.
Enabled: false

Index: 131
Name: Settings
Description: Adds settings menu to Twitch.
Enabled: true

Index: 132
Name: Unlock downloads
Description: Unlocks the ability to download any video. GIFs can be downloaded via the menu on long press.
Enabled: true

Index: 133
Name: Hide view count
Description: Hides the view count of Posts.
Enabled: false

Index: 134
Name: Dynamic color
Description: Replaces the default X (Formerly Twitter) Blue with the user's Material You palette.
Enabled: true

Index: 135
Name: Hide ads
Description: null
Enabled: true

Index: 136
Name: Hide recommended users
Description: null
Enabled: true

Index: 137
Name: Change link sharing domain
Description: Replaces the domain name of Twitter links when sharing them.
Enabled: true

Index: 138
Name: Open links with app chooser
Description: Instead of opening links directly, open them with an app chooser. As a result you can select a browser to open the link with.
Enabled: false

Index: 139
Name: Sanitize sharing links
Description: Removes the tracking query parameters from links before they are shared.
Enabled: true

Index: 140
Name: Promo code unlock
Description: Disables the validation of promo code. Any code will work to unlock all features.
Enabled: true

Index: 141
Name: Hide ads
Description: Hides all in-app ads.
Enabled: true

Index: 142
Name: Unlock pro
Description: Unlocks all pro features.
Enabled: true

Index: 143
Name: Hide ads
Description: Adds options to remove general ads.
Enabled: true

Index: 144
Name: Video ads
Description: Adds an option to remove ads in the video player.
Enabled: true

Index: 145
Name: Copy video URL
Description: Adds options to display buttons in the video player to copy video URLs.
Enabled: true

Index: 146
Name: Remove viewer discretion dialog
Description: Adds an option to remove the dialog that appears when opening a video that has been age-restricted by accepting it automatically. This does not bypass the age restriction.
Enabled: true

Index: 147
Name: Downloads
Description: Adds support to download videos with an external downloader app using the in-app download button or a video player action button.
Enabled: true

Index: 148
Name: Disable precise seeking gesture
Description: Adds an option to disable precise seeking when swiping up on the seekbar.
Enabled: true

Index: 149
Name: Seekbar tapping
Description: Adds an option to enable tap-to-seek on the seekbar of the video player.
Enabled: true

Index: 150
Name: Enable slide to seek
Description: Adds an option to enable slide to seek instead of playing at 2x speed when pressing and holding in the video player. Including this patch may cause issues with tapping or double tapping the video player overlay.
Enabled: false

Index: 151
Name: Seekbar thumbnails
Description: Adds an option to use high quality fullscreen seekbar thumbnails. Patching 19.16.39 or lower adds an option to restore old seekbar thumbnails.
Enabled: true

Index: 152
Name: Swipe controls
Description: Adds options to enable and configure volume and brightness swipe controls.
Enabled: true

Index: 153
Name: Disable auto captions
Description: Adds an option to disable captions from being automatically enabled.
Enabled: true

Index: 154
Name: Custom branding
Description: Applies a custom app name and icon. Defaults to "YouTube ReVanced" and the ReVanced logo.
Enabled: false

Index: 155
Name: Change header
Description: Applies a custom header in the top left corner within the app. Defaults to the ReVanced header.
Enabled: false

Index: 156
Name: Hide video action buttons
Description: Adds options to hide action buttons (such as the Download button) under videos.
Enabled: true

Index: 157
Name: Navigation buttons
Description: Adds options to hide and change navigation buttons (such as the Shorts button).
Enabled: true

Index: 158
Name: Hide player overlay buttons
Description: Adds options to hide the player cast, autoplay, caption button and next/ previous buttons.
Enabled: true

Index: 159
Name: Change form factor
Description: Adds an option to change the UI appearance to a phone, tablet, or automotive device.
Enabled: true

Index: 160
Name: Hide endscreen cards
Description: Adds an option to hide suggested video cards at the end of videos.
Enabled: true

Index: 161
Name: Disable fullscreen ambient mode
Description: Adds an option to disable the ambient mode when in fullscreen.
Enabled: true

Index: 162
Name: Hide layout components
Description: Adds options to hide general layout components.
Enabled: true

Index: 163
Name: Hide info cards
Description: Adds an option to hide info cards that creators add in the video player.
Enabled: true

Index: 164
Name: Hide player flyout menu items
Description: Adds options to hide menu items that appear when pressing the gear icon in the video player.
Enabled: true

Index: 165
Name: Disable rolling number animations
Description: Adds an option to disable rolling number animations of video view count, user likes, and upload time.
Enabled: true

Index: 166
Name: Hide seekbar
Description: Adds an option to hide the seekbar.
Enabled: true

Index: 167
Name: Hide Shorts components
Description: Adds options to hide components related to YouTube Shorts.
Enabled: true

Index: 168
Name: Disable suggested video end screen
Description: Adds an option to disable the suggested video end screen at the end of videos.
Enabled: true

Index: 169
Name: Hide timestamp
Description: Adds an option to hide the timestamp in the bottom left of the video player.
Enabled: true

Index: 170
Name: Miniplayer
Description: Adds options to change the in app minimized player.
Enabled: true

Index: 171
Name: Disable player popup panels
Description: Adds an option to disable panels (such as live chat) from opening automatically.
Enabled: true

Index: 172
Name: Remove player controls background
Description: Removes the dark background surrounding the video player controls.
Enabled: false

Index: 173
Name: Exit fullscreen mode
Description: Adds options to automatically exit fullscreen mode when a video reaches the end.
Enabled: true

Index: 174
Name: Open videos fullscreen
Description: Adds an option to open videos in full screen portrait mode.
Enabled: true

Index: 175
Name: Custom player overlay opacity
Description: Adds an option to change the opacity of the video player background when player controls are visible.
Enabled: true

Index: 176
Name: Return YouTube Dislike
Description: Adds an option to show the dislike count of videos with Return YouTube Dislike.
Enabled: true

Index: 177
Name: Wide searchbar
Description: Adds an option to replace the search icon with a wide search bar. This will hide the YouTube logo when active.
Enabled: true

Index: 178
Name: Shorts autoplay
Description: Adds options to automatically play the next Short.
Enabled: true

Index: 179
Name: Open Shorts in regular player
Description: Adds options to open Shorts in the regular video player.
Enabled: true

Index: 180
Name: SponsorBlock
Description: Adds options to enable and configure SponsorBlock, which can skip undesired video segments such as sponsored content.
Enabled: true

Index: 181
Name: Spoof app version
Description: Adds an option to trick YouTube into thinking you are running an older version of the app. This can be used to restore old UI elements and features. Patching 19.16.39 or lower includes additional older spoofing targets.
Enabled: true

Index: 182
Name: Change start page
Description: Adds an option to set which page the app opens in instead of the homepage.
Enabled: true

Index: 183
Name: Disable resuming Shorts on startup
Description: Adds an option to disable the Shorts player from resuming on app startup when Shorts were last being watched.
Enabled: true

Index: 184
Name: Theme
Description: Adds options for theming and applies a custom background theme (dark background theme defaults to amoled black).
Enabled: true

Index: 185
Name: Alternative thumbnails
Description: Adds options to replace video thumbnails using the DeArrow API or image captures from the video.
Enabled: true

Index: 186
Name: Bypass image region restrictions
Description: Adds an option to use a different host for user avatar and channel images and can fix missing images that are blocked in some countries.
Enabled: true

Index: 187
Name: Announcements
Description: Adds an option to show announcements from ReVanced on app startup.
Enabled: true

Index: 188
Name: Always repeat
Description: Adds an option to always repeat videos when they end.
Enabled: true

Index: 189
Name: Remove background playback restrictions
Description: Removes restrictions on background playback, including playing kids videos in the background.
Enabled: true

Index: 190
Name: Enable debugging
Description: Adds options for debugging.
Enabled: true

Index: 191
Name: Spoof device dimensions
Description: Adds an option to spoof the device dimensions which can unlock higher video qualities.
Enabled: true

Index: 192
Name: Check watch history domain name resolution
Description: Checks if the device DNS server is preventing user watch history from being saved.
Enabled: true

Index: 193
Name: GmsCore support
Description: Allows patched Google apps to run without root and under a different package name by using GmsCore instead of Google Play Services.
Enabled: true

Index: 194
Name: Bypass URL redirects
Description: Adds an option to bypass URL redirects and open the original URL directly.
Enabled: true

Index: 195
Name: Open links externally
Description: Adds an option to always open links in your browser instead of in the in-app-browser.
Enabled: true

Index: 196
Name: Remove tracking query parameter
Description: Adds an option to remove the tracking info from links you share.
Enabled: true

Index: 197
Name: Spoof video streams
Description: Spoofs the client video streams to fix playback.
Enabled: true

Index: 198
Name: Disable zoom haptics
Description: Adds an option to disable haptics when zooming.
Enabled: true

Index: 199
Name: Force original audio
Description: Adds an option to always use the original audio track.
Enabled: true

Index: 200
Name: Disable HDR video
Description: Adds an option to disable video HDR.
Enabled: true

Index: 201
Name: Remember video quality
Description: Adds an option to remember the last video quality selected.
Enabled: true

Index: 202
Name: Playback speed
Description: Adds options to customize available playback speeds, remember the last playback speed selected and show a speed dialog button in the video player.
Enabled: true

Index: 203
Name: Restore old video quality menu
Description: Adds an option to restore the old video quality menu with specific video resolution options.
Enabled: true

Index: 204
Name: Unlock premium
Description: null
Enabled: true
